preliz/internal/optimization.py

```diff
@@ -227,12 +227,15 @@ def optimize_beta_mode(lower, upper, tau_not, mode, dist, mass, prob):
             tau_not += 0.5 * tau_not
 
 
-def optimize_pymc_model(fmodel, target, draws, prior, initial_guess, bounds, var_info, p_model):
+def optimize_pymc_model(fmodel, targets, draws, prior, initial_guess, bounds, var_info, p_model):
```

Comment:
target is ok. as the target can be a mixture of distributuons

Reply:
Changed it to `target`